custom_cards/custom_card_saxel_fan/custom_card_saxel_fan.yaml

```diff
@@ -78,13 +78,13 @@ custom_card_saxel_fan_common:
   label: |-
     [[[
       if (entity.state != 'unavailable') {
-          if (variables.ulm_card_fan_slider_temp_attribute != False) {
+          if (variables.ulm_card_fan_slider_temp_attribute != false) {
               var temp = Math.round(entity.attributes[variables.ulm_card_fan_temp_attribute]);
               let temp_str = ' • ' + (temp ? temp : '0') + '°C';
           } else {
               let temp_str = ''
           }
-          if (variables.ulm_card_fan_slider_hum_attribute != False) {
+          if (variables.ulm_card_fan_slider_hum_attribute != false) {
```

Comment:
These should probably be null or empty checks. The attributes are strings

Reply:
Have fixed this - however moving forwards I am personally focussing on the new card_fan and leaving the saxel fan behind.